Gemfile.lock

```diff
@@ -557,7 +557,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.0.0)
     raindrops (0.13.0)
-    rake (10.4.2)
+    rake (10.5.0)
```

Comment:
Would be good to know if I need this update? It came with running 'bundle update debt_advice_locator' to update the version